Resources/Prototypes/Entities/Objects/Specific/Janitorial/soap.yml

```diff
@@ -103,3 +103,18 @@
   - type: StepTrigger
   - type: Item
     heldPrefix: omega
+
+- type: entity
+  name: ninja soap
+  id: SoapNinja
```

Comment:
This is more of a suggestion than a request.
Creating soap like this doesn't seem very futuristic ninja-y, maybe a soap projector would be better (i.e reviving #16927, addressing my review)

Reply:
honestly it just needs a cooler sprite to look holographic and high tech but thatll take time